### DISCUSSION TITLE
Password Change

### DISCUSSION THREAD

**Discussion by EnkerZan** [community/community#22457] [New to GitHub]
<p>I already took care of it, however this line I got in the email about it…</p>
<p>“We have determined that you are accessing GitHub with a password that is in a list of passwords commonly used on other websites.”</p>
<p>And just how in the hell would you know that? And for the record I have a different password for each site.</p>

**Comment by airtower-luna** [comment]
<p>Note that the message says “your password is a commonly used password”, not “you use that password somewhere else.” Security researchers can easily compile that kind of data from leaks that became public, and there are <em>a lot</em> of those. Attackers who try to guess passwords also use lists like that.</p>
<p>I recommend using a password manager and generating truly random, long passwords.</p>

**Comment by byrneh** [comment, marked as answer]
<aside class="quote no-group" data-username="EnkerZan" data-post="1" data-topic="238716">
<div class="title">
<div class="quote-controls"></div>
 EnkerZan:</div>
<blockquote>
<p>And just how in the hell would you know that? And for the record I have a different password for each site.</p>
</blockquote>
</aside>
<p>HI <a class="mention" href="/enkerzan">@EnkerZan</a> welcome to the community. This is not an uncommon question and has been answered a few times.<br>
I have provided a link below to a previous answer that will hopefully provide the answer/solution you are looking for in terms of explanation.<br>
<a href="https://github.community/t/does-github-keep-our-passwords-in-clear-text/228703/5">Checking for the use of compromised as described in NIST.SP.800-63b </a></p>

**Reply by eapo** [reply]
The referred thread doesn't exists anymore (would be nice to know the reason, or fix it), but there are some quotes here: https://stackoverflow.com/questions/68781632/how-does-github-knows-my-password-is-commonly-used-on-other-website

Would be nice to know more than that, because my pwd is unique, and want to know more about that list where it's listed.

**Reply by larryl79** [reply]
I want to know too where is listed my password...
